backend/tests/apps/mentorship/management/commands/test_mentorship_sync_module_issues.py

```
@@ -0,0 +1,208 @@
+        repo_label_map = cmd._build_repo_label_to_issue_map()
+
+        assert (10, "label-a") in repo_label_map
+        assert repo_label_map[(10, "label-a")] == {1, 4}
+        assert repo_label_map[(10, "label-b")] == {2}
+        assert repo_label_map[(20, "label-a")] == {3}
+        cmd.stdout.write.assert_any_call("Map built. Found issues for 3 unique repo-label pairs.")
+
+
+@patch("apps.mentorship.management.commands.mentorship_sync_module_issues.Task")
+@patch("apps.mentorship.management.commands.mentorship_sync_module_issues.Issue")
+def test_process_module_links_and_creates_tasks(MockIssue, MockTask, command):
```

Ruff (0.14.10)

Argument name `MockIssue` should be lowercase

(N803)